src/macros.jl

```diff
@@ -5,6 +5,9 @@
 
 using Base.Meta
 
+issum(s::Symbol) = (s == :sum) || (s == :∑) || (s == :Σ)
```

Comment:
My greek sucks. Please disregard. (Though `lowercase("∑") == "∑"` is still frustratingly evaluating to `true`)